canonical-web-and-design/maas-ui
# Remove legacy devices

## Done

- Remove the legacy code for managing devices.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the controllers list and controllers details, they should appear without errors.
- Go to the network tab for a controller and check that you can still edit interfaces.

## Fixes

Fixes: canonical-web-and-design/app-tribe#545.
